File: ass26/3.js
// 3. Write a program which accept string from user and copy capital
// characters of that string into another string.
// Input : “Marvellous Multi OS”
// Output : “MMOS”



function StrCpyCap(str)
{
    console.log("str : ",str);
    let upper ='';
    for(var i=0; i<str.length; i++)
    {
        if(str[i] == str[i].toUpperCase())
        {
            upper +=str[i]
        }
    }
    // upper = upper.replaceAll(' ','')
    return upper;

}
console.log(StrCpyCap('Marvellous Multi OS'))

File: ass24/3.py
# 3.Write a program which accept string from user and toggle the case.
# Input : “Marvellous Multi OS”
# Output : mARVELLOUS mULTI os

def strtogglex(s1):
    str1 = ''
    for i in s1:
        if(i.islower()):
            str1 = str1 + i.upper()
        else:
            str1 = str1 + i.lower()

    print("Upper case string is : ",str1)

def main():
    print("Enter String :")
    str1 = input()

    strtogglex(str1)


if __name__ == '__main__':
    main()